app/src/main/res/values/strings.xml

```diff
@@ -118,6 +118,8 @@
     <string name="restore_installing_tap_to_install">Tap to install</string>
     <string name="restore_next">Next</string>
     <string name="restore_restoring">Restoring backup</string>
+    <string name="restore_restoring_error_title">Not all apps restored</string>
+    <string name="restore_restoring_error_message">You can still try to re-install these apps later with "auto restore" turned on to get their data restored.</string>
```

Comment:
This is where apk install worked but app data restore failed right?

We could tweak this message a bit, it's already quite long but also sounds confusing.

Reply:
> This is where apk install worked but app data restore failed right?

This is shown after data restore when there are aps whose data could not be restored because they were not installed, but we have data in the backup for auto-restore later.

Suggestions for how to improve it are most welcome!

Reply:
@mikeNG thoughts?

Reply:
Does the user see which applications failed? If yes, then maybe this sounds better?

Unable to restore some apps
You can reinstall these apps manually and "Automatic restore" will attempt to restore their data.